tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -166,10 +176,13 @@ def completion_item_for_decl(
     )
 
 
+def location_to_location(loc) -> Location:
+    return Location("file://" + loc.path(), location_to_range(loc))
+
```

Comment:
this seems like something that should go in `chapel.lsp`

Reply:
It's left here deliberately. The reason is that I don't want to make the assumption in `chapel.lsp` that all Chapel files are `file://`-based. In CLS we do make that assumption, so that's where I put the function.